Khrystyna_Derenivska/src/app/app.component.html

```diff
@@ -0,0 +1,9 @@
+<!-- <app-new-item-add #appChild></app-new-item-add> -->
```

Comment:
Delete commented code before create a PR

Reply:
Deleted!

Reply:
Do I also need to delete tsconfig.spec.json file?